src/dev/Web/EastSeat.ResourceIdea.Web/Components/Pages/Backend/Tenants.razor

```diff
@@ -0,0 +1,395 @@
+                    </div>
+                </div>
+            }
+        </div>
+
+        @if (!filteredTenants.Any())
+        {
+            <div class="text-center py-5">
+                <i class="fas fa-building fa-3x text-muted mb-3"></i>
+                <h5 class="text-muted">No tenants found</h5>
+                <p class="text-muted">Try adjusting your search criteria or create a new tenant.</p>
+            </div>
+        }
+
+        <!-- Summary Statistics -->
+        <div class="row mt-5">
+            <div class="col-12">
+                <div class="card">
+                    <div class="card-header">
+                        <h5><i class="fas fa-chart-bar"></i> Tenant Statistics</h5>
+                    </div>
+                    <div class="card-body">
+                        <div class="row">
+                            <div class="col-md-3 text-center">
+                                <div class="stat-item">
+                                    <div class="stat-value text-primary">@allTenants.Count</div>
+                                    <div class="stat-label">Total Tenants</div>
+                                </div>
+                            </div>
+                            <div class="col-md-3 text-center">
+                                <div class="stat-item">
+                                    <div class="stat-value text-success">@allTenants.Count(t => t.Status == "Active")</div>
+                                    <div class="stat-label">Active Tenants</div>
+                                </div>
+                            </div>
+                            <div class="col-md-3 text-center">
+                                <div class="stat-item">
+                                    <div class="stat-value text-warning">@allTenants.Count(t => t.Status == "Trial")</div>
+                                    <div class="stat-label">Trial Tenants</div>
+                                </div>
+                            </div>
+                            <div class="col-md-3 text-center">
+                                <div class="stat-item">
+                                    <div class="stat-value text-info">@allTenants.Sum(t => t.UserCount)</div>
+                                    <div class="stat-label">Total Users</div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    }
+</div>
+
+@code {
+    private List<TenantListItem> allTenants = new();
+    private List<TenantListItem> filteredTenants = new();
+    private string searchTerm = string.Empty;
+    private string statusFilter = string.Empty;
+
+    protected override async Task OnInitializedAsync()
+    {
+        await ExecuteAsync(async () =>
+        {
+            await LoadTenants();
+        }, "Loading tenants");
+    }
+
+    private async Task LoadTenants()
```

Comment:
Reference the implementation of how the `GetTenantsListQuery` and `GetTenantsListQueryHandler` so that we don't have to simulate this implementation.